packages/core/src/app/payment/PaymentContext.tsx

```diff
@@ -19,6 +19,7 @@ export interface PaymentContextProps {
         schema: ObjectSchema<Partial<PaymentFormValues>> | null,
     ): void;
     hidePaymentSubmitButton(method: PaymentMethod, hidden?: boolean): void;
+    toggleLoadingOverlay?(isLoading: boolean): void;
```

Comment:
Does this need to be in the context? Can we implement this only in the component as this is used by only one component at this point?